js/templates/modals/about/donations.html

```diff
@@ -3,13 +3,15 @@
     <img src=qr>
   </div>
   <div id=donation-dashboard class="padMd flexCol">
-    <h2><%= ob.polyT('about.donationsTab.bannerCountDonations') %></h2>
+    <h2><%= ob.polyT('about.donationsTab.bannerCountDonations', { donationCountFloor: ob.donationCountFloor }) %></h2>
     <div class="padMd">
```

Comment:
The right side of the h2 should line up with the right side of the address and buttons. I'd take the padMd off of the address and buttons, and use a gutter class on the flexCol parent to handle the vertical space.